### ISSUE TITLE
Proposal: Rewards schema

### ISSUE THREAD

**Issue by WeaverStever** [schemaorg/schemaorg#3578]
A client had a rather rare and expensive guitar stolen from him several years ago and has offered a reward for its return.  I  am sitting down to brainstorm how to create a script that will expose the model and serial number to the internet, marking it as stolen property and therefore destroy any value in the resale market. 

- A Reward schema might also be useful for brand loyalty programs. (Concert tickets that include the latest CD etc.)

- The Reward schema might be useful for the investment community. I.E., rate of return over a period, cryptocurrency and NFT drops etc.   

Of course, there would also need to be a method to represent that stolen property had been recovered, perhaps a way to represent how many rewards were given in a brand loyalty program.

For instance, my cell phone provider (Helium Mobile) gives their cryptocurrency in their loyalty program.

**Comment by mfhepp** [comment]
This should actually pretty straightforward on the basis of the [GoodRelations conceptual model](http://wiki.goodrelations-vocabulary.org/Documentation/Conceptual_model) within schema.org (mind the [differences in naming](http://wiki.goodrelations-vocabulary.org/Cookbook/Schema.org)):


1. **Model the guitar as an actual product** with [schema:IndividualProduct](https://schema.org/IndividualProduct).
2. **Indicate that it is** a not just a schema:IndividualProduct [but **actually a guitar,** by using http://www.productontology.org/id/Guitar](http://www.productontology.org/id/Guitar) from <http://www.productontology.org>. Use a multi-typed entity for this.
3. Attach the **serial number** using [schema:serialNumber](https://schema.org/serialNumber). 
4. Attach the **model** as a string using [schema:model](https://schema.org/model).
5. **Model the legal ownership info** using [schema:owns](https://schema.org/owns) and [schema:OwnershipInfo](https://schema.org/OwnershipInfo); **omit [schema:ownedThrough](https://schema.org/ownedThrough) to indicate that the ownership is still upheld.**
6. **Define a custom [schema:BusinessFunction](https://schema.org/BusinessFunction) for "ReturnToOwner" or "Bring back stolen item" or similar.** The meaning of this business function (which is bundle of rights) would be "provide possession of something that I already own legally"*. ("Sell" legally means: provide ownership and possession", "Lease out" would mean provide temporary possession ("access"), but no ownership.)
7. **Indicate that you are looking for *someone to bring back the stolen guitar and are willing to pay a reward for its return.***
For this, use [schema:Demand](https://schema.org/Demand), link to your "ReturnToOwner" business function via [schema:businessFunction](https://schema.org/businessFunction).
8. **Link from the demand node to the guitar entity** via [schema:itemOffered](https://schema.org/itemOffered) (or via [schema:](https://schema.org/includesObject) - so you say *"I am looking for someone who can provide ownership on this guitar, which I own."*. You can **model the reward** using [schema:priceSpecification](https://schema.org/priceSpecification), **which actually defines the compensation you are offering if someone accepts your request (the demand entity)**.

The principles is illustrated in this figure (schema:Offer would be schema:Demand in here) ![GoodRelations conceptual model](http://wiki.goodrelations-vocabulary.org/File:Gr-conceptual-model2.png) 

Hope that helps :-)!

And you can see very nicely that the [GoodRelations conceptual model from 2008 fits to a surprisingly broad range of scenarios ***without a lot of tweaking***](https://www.heppnetz.de/projects/goodrelations/).


A bit more of background regarding the modeling of ownership information (e.g. for portable shopping histories) is in the following paper:

László Török and Martin Hepp: *Towards Portable Shopping Histories: Using GoodRelations to Expose Ownership Information to E-Commerce Sites.* [ESWC 2014](https://dblp.org/db/conf/esws/eswc2014.html#TorokH14): 691-705.
A free PDF is available from https://www.heppnetz.de/files/portable-shopping-histories-eswc2014.pdf

```bibtex
@inproceedings{DBLP:conf/esws/TorokH14,
  author       = {L{\'{a}}szl{\'{o}} T{\"{o}}r{\"{o}}k and
                  Martin Hepp},
  editor       = {Valentina Presutti and
                  Claudia d'Amato and
                  Fabien Gandon and
                  Mathieu d'Aquin and
                  Steffen Staab and
                  Anna Tordai},
  title        = {Towards Portable Shopping Histories: Using GoodRelations to Expose
                  Ownership Information to E-Commerce Sites},
  booktitle    = {The Semantic Web: Trends and Challenges - 11th International Conference,
                  {ESWC} 2014, Anissaras, Crete, Greece, May 25-29, 2014. Proceedings},
  series       = {Lecture Notes in Computer Science},
  volume       = {8465},
  pages        = {691--705},
  publisher    = {Springer},
  year         = {2014},
  url          = {https://doi.org/10.1007/978-3-319-07443-6\_46},
  doi          = {10.1007/978-3-319-07443-6\_46},
  timestamp    = {Tue, 20 Aug 2019 15:27:53 +0200},
  biburl       = {https://dblp.org/rec/conf/esws/TorokH14.bib},
  bibsource    = {dblp computer science bibliography, https://dblp.org}
}
```

**Comment by chaals** [comment]
> A client had a rather rare and expensive guitar stolen from him several years ago and has offered a reward for its return. I am sitting down to brainstorm how to create a script that will expose the model and serial number to the internet, marking it as stolen property and therefore destroy any value in the resale market.

Note that there is a pretty clear problem here - if I decide I quite like a particular car (or painting, or other identifiable object) I see, that isn't sufficient reason for me to be able to destroy it's resale value. Likewise, noting that I am looking for someone to provide me with that car, in exchange for a reward, is viewed in different ways by different people. It seems like a mechanism for market manipulation that is generally subject to regulatory control (Laws about securities, misrepresentation, fraud, theft, incitement to criminal acts...). Many people consider some of these laws quite important.

On a blockchain there is a general mechanism for deciding if someone owns a token (roughly, possession - as demonstrated by being able to control the disposition of the token - is 10 tenths of "the law"). In most legal systems, that's insufficient; blockchains are useful to register evidence of ownership, but in general assuming they are sources of "the Truth" is to misunderstand how "The Law" views Truth.

**Comment by mfhepp** [comment]
Hi @chaals:

Note that my reply focusses on how you can express the statement:

*Person X is the legal owner of the guitar with the serial number Y, model ACME123. X seeks people or organizations that can provide possession on this guitar and X promises to pay a reward of Z in that case.*

What my sketch does not model is that the item is marked as stolen.

For that, one could think along the following lines, IMO:

1. product -> [schema:subjectOf](https://schema.org/subjectOf) -> a description of the theft modeled as a [schema:Event](https://schema.org/Event).
2. product -> [schema:itemCondition](https://schema.org/itemCondition) -> a new [schema:OfferItemCondition](https://schema.org/OfferItemCondition) e.g. ***schema:StolenCondition.***.

Please also see that my proposed representation is agnostic as to the means and target audience of the statement.
One could exclude law enforcement from the compensation, or one could constrain the validity of the statement to legal means, e.g. via [schema:eligibleCustomerType](https://schema.org/eligibleCustomerType).

**Comment by chaals** [comment]
@mfhepp 

> What my sketch does not model is that the item is marked as stolen

Sure. I think you made it clear that schema.org as is provides a really rich framework for descriptions of real things and ideas. (I suggest this is largely because it is fundamentally based on RDF as a way to represent things).

I think the "bigger picture" I tried to point to is more about what are the limits of what we can, or should be able to, accomplish with schema.org. I don't think they are in its descriptive power, so much as in the ways we can use that, and particularly what happens when we start processing the information automatically. (With apologies to @WeaverStever and all, because I think I am quickly getting off the track this particular issue is trying to follow).

**Comment by mfhepp** [comment]
> (I suggest this is largely because it is fundamentally based on RDF as a way to represent things).

The point I was trying to make is that it is not the plain RDF-like meta-model in schema.org that allows this, but the **Agent-Promise-Object-Compensation** model of GoodRelations ;-).

**Comment by WeaverStever** [comment]
@mfhepp 

Thank you for pointing out the GoodRelations diagram and outline.

@chaals 

Yes, market manipulation might be a problem in the case of an offer/reward for stolen merchandise.  A simple solution would be to  provide a method to prove the attestation that the musical instrument is stolen.  A PDF or image of the police report would be sufficient.  The potential buyer would be informed that there is a claim against the stolen item and simply call the police department.  

Basically, I'm looking for a Rich Snippet result that displays in search when the model and serial number are queried, stating that the instrument was reported stolen on a specific date.   

In the case of a motor-vehicle, there are official databases, titles cannot be transferred, nor can a stolen vehicle be insured.

As for the loyalty program (Offer/action/reward) side of the discussion, there is a discussion here. #3563

**Comment by mfhepp** [comment]
> Basically, I'm looking for a Rich Snippet result that displays in search when the model and serial number are queried, stating that the instrument was reported stolen on a specific date.
> 
> In the case of a motor-vehicle, there are official databases, titles cannot be transferred, nor can a stolen vehicle be insured.

@danbri @alex-jansen @tmarshbing - could you maybe point relevant groups to this discussion? I think that a "stolen item alert" feature (or "missing items", maybe even "missing people" in a similar fashion) would be a useful, straightforward feature for search engines, along the lines of

- markup for describing stolen / missing items with schema.org
    - as sketched above, requires likely just one or two additional elements plus an example
    - don't forget to add contact details using schema:Person etc. in the examples
- some useful search engine feature, e.g.
    - an email alert
    - a warning if the item appears in a product rich snippet ("this item has been reported lost or stolen, (link)".

Looks like a quick win and useful innovation to me.

**Comment by chaals** [comment]
@mfhepp 

> a "stolen item alert" feature (or "missing items", maybe even "missing people" in a similar fashion) would be a useful, straightforward feature for search engines

I think it's a feature that could be really dangerous (even more so in the case of people). Without a really solid fact-checking system the potential for abuse is massive and I think it is a safe assumption that it would be realised. Governments and police forces are known to make pretty controversial claims about things (some more so than others, but squeaky clean isn't something I have seen), so how you set up such a system is likely to be anything but straightforward for any search engine operating on a more or less global scale.

As @WeaverStever noted, there are registries of motor vehicles and ownership, and they can often be consulted. Likewise, police services can decide to publish other ergistries of stolen goods, calls for missing people, etc. But while I haven't worked for a search engine for many years, I would hope they don't rush into something like this, because I fear the consequences if it isn't *very* carefully thought through.

**Comment by WeaverStever** [comment]
Unlike stolen vehicles, police generally do not publish information about stolen goods. I think a copy of a police report, linked in the rich snippet, would be enough for the potential buyer to discern if they want to risk making the purchase.

As for fale or misleading entries, that perp would have to know the model number and serial number. Without the identifiers, the specific search would not yield an exact hit.

There are some (very dated) websites that try to present a registry of stolen instruments,  but that seems too labor intensive considering the current search technology.

The desired outcome is that the potential  buyer informs the thief of the reward and advises the thief to return the stolen item(s) for the reward.

**Comment by chaals** [comment]
@WeaverStever 

> The desired outcome is that the potential buyer informs the thief of the reward and advises the thief to return the stolen item(s) for the reward.

Maybe. But if the outcome that gets realised is a well-ordered efficient market in rewards for stolen goods, and a boom in the market for theft-protection, we might not end up better off.

I think another desired outcome might be a reduction in the market value of stolen goods, and thus the incentive to steal them. And another might be that your client gets their guitar back.

And this proposal might be the right approach to achieve outcomes we want, or represent the best set of trade-offs we can find. I think it's valuable to think as far as we can along the consequences. It helps understand some of the things that feed into whether a search engine will decide to do something or not.

**Comment by alex-jansen** [comment]
The stolen product discussion seems to be not so much about the data modeling part itself, which already is well-covered by the examples provided by @mfhepp . The bigger issue here (and independent from structured data) is trust: It makes a big difference if stolen data are published by a recognized authority on a trusted domain (think about Interpol, [NMPR](https://thenmpr.com/about), Enquirus, and others) or on a random less-known website.

A related discussion can be found under #3230, about verifiability of product certifications. Product certifications published by the EC, for example as part of the EPREL database, can be trusted. Certifications published with an associated W3C verifiable credential can be trusted. Certifications published on a merchant website without any means of verifying would be much less trustworthy.

Also related to this topic is #3229, which proposes the addition of a schema for product recalls. Recalls are typically published by trusted authorities, such as the FDA (US), Rapex (European Commission) and OECD. Consuming systems would typically only trust data published by such authorities.
